### ISSUE TITLE
MAS 1.4.3: Elements must have sufficient color contrast (div:nth-child(1))

### ISSUE THREAD

**Issue by priysn** [Microsoft/BotFramework-WebChat#1354]
**Issue Details**
Elements must have sufficient color contrast
color-contrast

**Repro Steps:**
1.	Open URL [](https://microsoft.github.io/BotFramework-WebChat/full-bundle/) in Edge browser.
2.	Notice 'Upload File' and 'Send' button icons present in bottom of screen.
3.	Type 'Help' in the input control at the bottom of the screen and hit enter (or click on send button).
4.	Content will upload, Navigate to " Demo for GitHub repository sample " button and click on it.
5.	‘sample:github-repository’ page appear in bottom section.
6.	Verify the color contrast on ‘No renderer for this attachment’ text.

**Element Path**
li:nth-child(1) > .css-djy7c2 > .css-19kadhh > div:nth-child(1)

**Snippet**
<div>No renderer for this attachment</div>

**How to fix**
Fix any of the following:
- Element has insufficient color contrast of 3.99 (foreground color: #ffffff, background color: #ff0000, font size: 12.0pt, font weight: normal). Expected contrast ratio of 4.5:1

**Test Environment:**
OS: Windows 10
OS Build: 17763.55
OS Version: 1809
Browser: Edge
Tool: K4W

This accessibility issue was found using Keros for Web, a tool that helps debug and find accessibility issues earlier. Get more information & download this tool at [](http://aka.ms/K4Web.)

![mas1 4 3_sample](https://user-images.githubusercontent.com/41668935/48478360-10a02000-e82a-11e8-90b5-acdb04a70ce7.png)

**Comment by compulim** [comment]
@corinagum please change it to `#EF0000`.

**Comment by corinagum** [comment]
Closed as duplicate of #1341

**Comment by cwhitten** [comment]
#HCL;#Accessibility;#HCL_BotFramework_WebChat;A11yMAS;WCAG1.4.3
